app/build.gradle

```diff
@@ -34,8 +28,29 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
```

Comment:
Why 8 vs 11?

Reply:
This has to do with the lowest Java version that the app is compatible with.  Changing this value may restrict the number of devices the app can support.  If we need to use Java 11-specific features we can change this value, but best to leave as is for now.